Rakefile

```diff
@@ -10,12 +10,5 @@ require "ci/reporter/rake/minitest" if Rails.env.test?
 
 Whitehall::Application.load_tasks
 
-begin
-  require "rubocop/rake_task"
-  RuboCop::RakeTask.new
-rescue LoadError
-  # Rubocop isn't available in all environments
-end
-
```

Comment:
Why don't we need this anymore?

Reply:
We call rubocop through a shell command `sh("bundle exec rubocop")` rather than using the rask task rubocop provides.